### ISSUE TITLE
impossible to install type for react-redux 7.x.x

### ISSUE THREAD

**Issue by RenaudJenny-TomTom** [flow-typed/flow-typed#4052]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**
**If the current behavior is a bug, please provide the steps to reproduce.**

I want to type `react-redux` v7. When I do
```
$ flow-typed install react-redux@7.2.0
```

I got

```
• Searching for 1 libdefs...
• rebasing flow-typed cache...
!! No flow@v0.122.0-compatible libdefs found in flow-typed for the explicitly requested libdefs. !!

Consider using `flow-typed create-stub react-redux@7.2.0` to generate an empty libdef that you can fill in.
```

It's quite bizarre as the lib def exist, especially that if I'm using the `search` function, it works exactly as expected:

```
$ flow-typed search react-redux      
• rebasing flow-typed cache...done.

Found definitions:
╔═════════════╤═════════════════╤══════════════════════╗
║ Name        │ Package Version │ Flow Version         ║
╟─────────────┼─────────────────┼──────────────────────╢
║ react-redux │ v7.x.x          │ v0.104.x-0.141.x     ║
╟─────────────┼─────────────────┼──────────────────────╢
║ react-redux │ v7.x.x          │ >=v0.142.x           ║
╟─────────────┼─────────────────┼──────────────────────╢
║ react-redux │ v7.x.x          │ >=v0.89.x <=v0.103.x ║
╟─────────────┼─────────────────┼──────────────────────╢
║ react-redux │ v6.x.x          │ >=v0.89.x <=v0.103.x ║
...
```

**What is the expected behavior?**

`react-redux` types are installed.

**If this is a feature request, what is motivation or use case for changing the behavior?**

-

**Local Environment Information**
If this issue is specific to the CLI, provide the results of running `npx envinfo` from the command line in a GitHub gist.

**Comment by Brianzchen** [comment]
Sorry this is my fault. Are you on version `0.104.x - 0.141.x`? Let me raise a fix for this

**Comment by RenaudJenny-TomTom** [comment]
@Brianzchen yes I'm exactly on this version (we're currently using Flow `0.122.0`).
I thought the problem could come from our environment, but that's good news (for us at least 😂).

Thanks for taking care of this.

**Comment by GAntoine** [comment]
Fixed in #4053, thanks for the PR @Brianzchen

**Comment by RenaudJenny-TomTom** [comment]
I confirm it works, thanks a lot for the quick fix!
